### PR TITLE
Remove python package from Ubuntu22 default packages, align stunnel test name with our current pattern

### PR DESCRIPTION
**Pull request by dreambeyondorange**
### Description of changes
* `Python` is no longer a package in canonical Ubuntu, so removed from default packages in Ub22

### Tests
* Ran kitchen tests that were trying to install python, so far they have succeeded.

### Checklist
- [ ] Make sure you are pointing to **the right branch** and add a label in the PR title (i.e. **2.x** vs **3.x**)
- [ ] Check all commits' messages are clear, describing what and why vs how.
- [ ] Make sure **to have added unit tests or integration tests** to cover the new/modified code.
- [ ] Check if documentation is impacted by this change.

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: kitchen.resources-install.yml [hunk 1]
```diff
@@ -133,7 +133,7 @@ suites:
         - recipe:aws-parallelcluster-platform::directories
         - resource:package_repos
         - resource:install_packages
-  - name: stunnel_installed
+  - name: stunnel
```
Comment (awslitvit) [inline kitchen.resources-install.yml:136]: Nitpick. These changes are unrelated, I'd suggest to submit 2 separate commits.

**Review by awslitvit — APPROVED** [review]

File: kitchen.resources-install.yml (hunk 1 above)
Comment (dreambeyondorange) [reply kitchen.resources-install.yml:136]: will do going forward
